tracer/test/Datadog.Trace.TestHelpers/EnvironmentHelper.cs

```diff
@@ -147,6 +152,18 @@ public static string GetProfilerPath()
             return path;
         }
 
+        public static string GetIntegrationsJsonFilePath()
+        {
+            string fileName = "integrations.json";
+            var path = Path.Combine(GetTracerHomePath(), fileName);
+            if (!File.Exists(path))
+            {
+                throw new Exception($"Attempt 3: Unable to find integrations at {path}");
```

Comment:
I guess you had attempts 1 and 2 at some point 😅

Reply:
Old code dealing with differences in local dev environments and CI.